docs/blog/newsletter-november-2021.rst

```diff
@@ -0,0 +1,81 @@
+
+.. post:: November 03, 2021
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – November 2021
+#########################################
+
+Hi folks! November has somehow rolled around (already??) and it's that newsletter time of the month again.
+
+As promised last time: I'm super excited to let you know that the `Salary Survey is now live <https://salary-survey.writethedocs.org/>`__. It takes less than ten minutes to complete, so please consider filling it out - you'll be contributing to a really valuable resource for documentarians. All responses are completely anonymous, and we're very careful to not to publish any personally identifiable information (given the small size of the community). If you're up for it, `fill it out here <https://salary-survey.writethedocs.org/>`__.
+
+Plus, great news from down under - we've just announced the Australia and India meetup will be on TODO date. This "super-meetup" will be a two-hour event with two fab speakers, and will be free to attend. `Sign up for the event here <TODO link>`__!
+
+And with that, let's take a look at what everyone's been discussing on Slack...
+
+------------------------------------------------
+Managing when your manager lacks docs experience
+------------------------------------------------
+
+This month, our documentarians discussed how they have handled differences with a manager who has no experience with documentation. Among those who have worked for managers without a docs background, there were roughly two buckets: the manager either deferred to the technical writer's expertise or did not.
+
+If your manager is in the first category, you'll likely encounter less heavyhanded oversight, and docs-related disagreements may be rare. If your manager in the second category, you have a harder path to tread, especially when your manager's perspective on docs-related tasks doesn't quite match yours.
```

Comment:
At first read I was confused by the other meaning of *oversight*, how about an alternative?

Reply:
went for supervision